### ISSUE TITLE
Desktop Safari pinch() bugged

### ISSUE THREAD

**Issue by CrackHD** [davidfig/pixi-viewport#108]
With sample code from here: https://davidfig.github.io/pixi-viewport/jsdoc/

With Mobile Safari, everything is fine (like every other browser).
Although Desktop Safari is bugged (tested on latest 12.0.2). If you zoom in and zoom out the same region (with mac touchpad) for a few times, render visuals get damaged (image "blurring") and you have to fire Command+0 to reset zoom (View -> Actual Size).

`.pinch()` thing actually seem to work in parallel with native browser zoom.

Anyone acknowledged of this situation? Any workarounds?

**Comment by davidfig** [comment]
Yup. Safari uses the non-standard Gesture API to handle pinches on the trackpad. Sigh. I'm working on adding support for this.

**Comment by dylanpyle** [comment]
If anyone else is running into this need, we just open-sourced https://github.com/ca-la/pixi-viewport-gesture-pinch as a lightweight solution. Thanks for all your hard work on this @davidfig!

**Comment by davidfig** [comment]
Cool. I'll take a look at the code and see if I can incorporate it into pixi-viewport. I have need of this for my current project so the work should be done over the next couple of weeks.
